File: java/P001.java
/*
 * Finds the sum of all multiples of 3 or 5 below 1000. 
 * Code taken from nayuki as a template to work from.
 */
public final class P001 implements Solution {

  public static void main(String[] args) {
    System.out.println(new P001().run());
  }

  public String run() {
    int sum = 0;
    for (int i = 0; i < 1000; i++) {
      if (i % 3 == 0 || i % 5 == 0)
        sum += i;
    }
    return Integer.toString(sum);
  }
}

File: python/P001.py
def P001()->int:
    """Finds the sum of all multiples of 3 or 5 below 1000. """
    tot = 0
    for i in range(1000):
      if (i % 3 == 0 or i % 5 == 0):
        tot += i;
    return tot

if __name__=='__main__':
    P001()
